corehq/apps/export/models/new.py

```diff
@@ -148,13 +148,15 @@ def create_default_from_export_item(group_schema_path, item, app_ids_and_version
         """
 
         is_main_table = group_schema_path == MAIN_TABLE
+        is_advanced = isinstance(item, SystemExportItem) and item.is_advanced
```

Comment:
hmm i see why it's necessary to store the `is-advanced` on the export item, but technically that kind of breaks our idea of having `ExportItem` just be the data and not the presentation. wonder if there's any solution to that